src/lib/FieldType/Mapper/UserAccountFieldValueFormMapper.php

```diff
@@ -46,22 +46,22 @@ public function mapFieldValueForm(FormInterface $fieldForm, FieldData $data)
         $formConfig = $fieldForm->getConfig();
         $rootForm = $fieldForm->getRoot()->getRoot();
         $formIntent = $rootForm->getConfig()->getOption('intent');
-        $isTranslation = $rootForm->getData() instanceof ContentTranslationData;
+        $formData = $rootForm->getData();
+        if ($formData instanceof ContentUpdateData) {
+            return;
+        }
+
         $formBuilder = $formConfig->getFormFactory()->createBuilder()
             ->create('value', UserAccountFieldType::class, [
                 'required' => true,
                 'label' => $fieldDefinition->getName(),
                 'intent' => $formIntent,
                 'constraints' => [
-                    new UserAccountPassword(['contentType' => $rootForm->getData()->contentType]),
+                    new UserAccountPassword(['contentType' => $formData->contentType]),
                 ],
             ]);
 
-        if ($isTranslation) {
-            $formBuilder->addModelTransformer($this->getModelTransformerForTranslation($fieldDefinition));
```

Comment:
`getModelTransformerForTranslation` method should be deprecated as it's not used anymore.

Reply:
@adamwojs could you, please, take a look at https://github.com/ezsystems/ezplatform-content-forms/pull/61#discussion_r927436324? This PR has to be put on hold until we decide what to do with content translating, as discussed:
Creating - `userCreateData`  (instead of `contentCreateData`)
Updating - `userUpdateData` (instead of `contentUpdateData`)
Translating - `contentUpdateData` (standard across all contents)

Dawid says we should use `contentTranslateData` system-wise but we have not been using it at all anywhere. Could you please share your point of view on this?